### ISSUE TITLE
Tooltips on the buttons that explain what they do

### ISSUE THREAD

**Issue by voidptr** [rollerderby/scoreboard#405]
Since the start/stop jam and TO, etc, buttons now change titles, it can be hard to remember what does what when setting up shortcuts. To ease the transition, I'd like to see a mouse-over tooltip over the buttons that describes what they do. 

Similar tooltips on all the other buttons would be neat too.

**Comment by brian-brazil** [comment]
What they do changes over time, and we don't use tooltips elsewhere. There's also a possibility that the tooltip would get in the way during actual gameplay.

**Comment by frank-weinberg** [comment]
I can see how tooltips might be useful for new users. But doing them in a way that doesn't interfere with operation could be tricky and in order to keep the interface consistent they should probably be done for at least all buttons on a tab if not across the whole UI which makes this a rather large task. (Especially figuring out good texts may easily be more work than implementing the tooltips.)

**Comment by charlie-chaperon** [comment]
I agree that it could be useful for new score board operators. Maybe you could add a button in the top line "turn tooltips on/off" which is deactivated by default? That way you can also use it to quick check / remember how a (rarely used) button works.

**Comment by gdunn** [comment]
It's easy enough to add a title attribute to where we want tool tips. The hard part will be enabling/disabling it. Would need to feed in an option (or options) along with the gameid
![Screenshot 2022-07-24 174413](https://user-images.githubusercontent.com/1120941/180637563-6f531af9-bc63-4317-a177-fe2983fdbaff.png)
